test/integration/index.js

```diff
@@ -566,8 +566,8 @@ describe('bootstrap()', () => {
         .set('Cookie', ['myCookie=1234'])
         .expect((res) => {
           const csp = getHeaders(res, 'content-security-policy');
-          csp['img-src'].should.include('http://www.google-analytics.com/collect', 'bar', 'self', 'data:');
-          csp['script-src'].should.include('http://www.google-analytics.com/analytics.js', 'poo', 'self');
+          csp['img-src'].should.include('www.google-analytics.com', 'bar', 'self', 'data:');
+          csp['script-src'].should.include('www.google-analytics.com', 'poo', 'self');
```

Comment:
This test is probably not testing what you think it is. There's a typo `foo` => `poo` which *should* fail the test, but currently is not.